workflows/crisprseq_screening.nf

```diff
@@ -174,14 +182,26 @@ workflow CRISPRSEQ_SCREENING {
         .set { ch_counts }
     }
 
-
     if(params.crisprcleanr) {
-        ch_crispr_normalize = Channel.of([id: "count_table_normalize"])
-        CRISPRCLEANR_NORMALIZE(
-            ch_crispr_normalize.concat(ch_counts,ch_crisprcleanr).collect(),
-            params.min_reads,
-            params.min_targeted_genes
-        )
+        ch_crispr_normalize = Channel.of([id: "count_table_normalize"]).concat(ch_counts)
+
+        if(params.crisprcleanr.endsWith(".csv")) {
+            CRISPRCLEANR_NORMALIZE(
+                ch_crispr_normalize.collect(),
+                '',
+                ch_crisprcleanr_file,
+                params.min_reads,
+                params.min_targeted_genes
+        ) } else
+        {
+            ch_crispr_normalize = Channel.of([id: "count_table_normalize"]).concat(ch_counts)
+            CRISPRCLEANR_NORMALIZE(
```

Comment:
Have you checked if you get a warning complaining about a module used twice? You may need to import the module twice, one with an alias.

Reply:
No it doesn't complain